simulators/nav/src/components/field/Field.vue

```diff
@@ -347,22 +370,51 @@ export default class Field extends Vue {
         break;
       }
 
+      case FieldItemType.MOVE_ROVER: {
+        this.moveRover(clickOdom);
+        break;
+      }
+
       /* no default */
     }
-  } /* addFieldItem() */
+  } /* handleFieldClick() */
 
-  /* Download a test case in JSON format with the current field items on the
-     canvas. */
-  private downloadTestCase():void {
-    const testCaseData:string = JSON.stringify(this.fieldState, null, 2);
-    const dataToDownload = `data:text/plain;charset=utf-8,${encodeURIComponent(testCaseData)}`;
+  /* When field is double-clicked, perform action corresponding to the current
+     draw mode. */
+  private handleFieldDblClick(event:MouseEvent):void {
+    const clickLoc:Point2D = {
+      x: event.offsetX * PIXEL_DENSITY,
+      y: event.offsetY * PIXEL_DENSITY
+    };
+    const clickOdom:Odom = canvasToOdom(clickLoc, this.canvasHeight, this.scale,
+                                        this.fieldCenterOdom);
 
-    this.dataToDownload.setAttribute('href', dataToDownload);
-    this.dataToDownload.setAttribute('download', `test_case_${new Date().toJSON()}.json`);
-    this.dataToDownload.click();
-    this.dataToDownload.removeAttribute('href');
-    this.dataToDownload.removeAttribute('download');
-  } /* downloadTestCase() */
+    switch (this.drawMode) {
+      case FieldItemType.MOVE_ROVER: {
+        this.moveStartOdom(clickOdom);
+        break;
+      }
+
+      /* no default */
```

Comment:
Is there no default because the default is to do nothing or is a drawMode that reaches the default case undefined behavior?

Reply:
The default is to do nothing. The reason for this comment is readability: https://eslint.org/docs/rules/default-case